### PR TITLE
feature/typed-reducer: Exported reducer has type.

### PR DESCRIPTION
**Pull request by robinMcA**
Auth reducer was being imported with the type any. The reducer will by
typed in inports

### CODE REVIEW

File: lib/tsconfig.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "sourceMap": true,
     "importHelpers": true,
     "skipLibCheck": true,
+    "declaration": true,
```
Comment (haolinj) [inline lib/tsconfig.json]: I think `declaration` does not compatible with `allowJs` and `isolatedModules`. Is rollup using `tsconfig.rollup.json`?

Comment (robinMcA) [reply lib/tsconfig.json]: I've removed it in b4c8c85, I had forgotted that I'd added it

**Comment by haolinj** [comment]
Thanks @robinMcA

**Pull request merged**